packages/apps/plugins/plugin-thread/src/ThreadPlugin.tsx

```diff
@@ -190,14 +190,14 @@ export const ThreadPlugin = (): PluginDefinition<ThreadPluginProvides> => {
         resolver: (intent) => {
           switch (intent.action) {
             case ThreadAction.CREATE: {
-              return { object: new ThreadType() };
+              return { data: new ThreadType() };
             }
 
             case ThreadAction.SELECT: {
               state.threads = intent.data?.threads;
               state.active = intent.data?.active;
               state.focus = intent.data?.focus;
-              break;
+              return { data: true };
```

Comment:
pre previous. seems a bit odd to return true.
if it's just success then the absence of an error would be ok?
and we can standardize a way to return errors? or just `return { err };`

Reply:
data is optional so strictly this could also just be `{}`, it just needs to return something truthy

Reply:
currently the type is `error` vs `err` but yes, that's the expected way to return errors (and the chain will terminate and return if an error is returned in the middle)